test/test_commands_serve.rb

```diff
@@ -76,15 +76,16 @@ def custom_opts(what)
 
       should "keep config between build and serve" do
         custom_options = {
-          'config'  => ['_config.yml', '_development.yml'],
-          'serving' => true,
-          'watch'   => false # for not having guard output when running the tests
+          "config"  => ["_config.yml", "_development.yml"],
```

Comment:
This should be `%w(_config.yml _development.yml)` This is a bug in our `.rubocop.yml`